platform/src/bridge.ts

```diff
@@ -30,37 +30,19 @@ export const run = async (
     console.log("Initing output file at", outputPath);
     await Bun.write(outputPath, "");
 
-    // const proc = Bun.spawn(
-    //   [
-    //     "poetry",
-    //     "run",
-    //     "python",
-    //     "services/entry.py",
-    //     scriptName,
-    //     JSON.stringify(args),
-    //   ],
-    //   {
-    //     onExit: async (proc, exitCode, signalCode, error) => {
-    //       // exit handler
-    //       const result = Bun.file("out.json");
-    //       const text = await result.text();
-    //       resolve(JSON.parse(text));
-    //     },
-    //   }
-    // );
-
-    // Use nodejs spawn
-    // I seem to have to use this because the bun stream doesn't work with readline
-
     const proc = spawn(
       "poetry",
       [
         "run",
         "python",
         "services/entry.py",
+        "--service",
```

Comment:
Can we make script name required and position? I think the absolute minimum here is `python services/entry.py <service>`, and everything else is probably technically optional (even if we usually use it)